### PR TITLE
Implementation of footer component

### PR DESCRIPTION
**Pull request by rashidyaqoob**

### CODE REVIEW

File: components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import Menu from '../components/Menu'
+import Image from 'next/image'
+import footerStyles from '../styles/Footer.module.css'
+import { FiLinkedin, FiTwitter, FiFacebook} from 'react-icons/fi';;
+
+const Footer = () => {
+	return (
```
Comment (AlviMurtaza) [inline components/Footer.js]: let's use 2 spaces throughout all the files

File: components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+					<div className={footerStyles.socialLinks}>
+						<Menu title='Social Sharing' links={[{ link: '/a', text: <FiFacebook /> }, { link: '/a', text: <FiTwitter /> }, { link: '/a', text: <FiLinkedin /> }]} />
+					</div>
+					<div className={footerStyles.footerLogo}>
+						<Image src='/svg/axl-logo.svg' alt='Axelerant Logo' width={134} height={100} className={footerStyles.footerLogo}></Image>
+					</div>
+				</div>
+			</div>
+			<div className={footerStyles.footerSecondary}>
+				<p>© 2020 Axelerant. All Rights Reserved.</p>
```
Comment (AlviMurtaza) [inline components/Footer.js]: use `&copy;`

File: components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+			</div>
+		</footer>
+		);
+}
+ 
+export default Footer;
```
Comment (AlviMurtaza) [inline components/Footer.js]: Add EOL

File: components/Menu.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import footerStyles from '../styles/Footer.module.css'
+
+export default function Menu({ title, links }) {
+	return (
+		<div className={footerStyles.menu}>
+			<p className={footerStyles.menuTitle}>{title}</p>
+			<ul className={footerStyles.menuLinkList}>
+				{links.map(({ link, text }) => <li key={text} className={footerStyles.menuLinkItem}><a className={footerStyles.menuLink} href={link}>{text}</a></li>)}
```
Comment (AlviMurtaza) [inline components/Menu.js]: - use 2 space indentation.
- Break this into multiple lines.
- Add EOL

**Comment by skippednote** [comment]
IMG_0063
The footer looks broken,

File: components/Footer.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
+import Menu from '../components/Menu'
+import Image from 'next/image'
+import footerStyles from '../styles/Footer.module.css'
```
Comment (skippednote) [inline components/Footer.js:3]: Please organise the imports.

**Pull request merged**